reporters_db/data/reporters.json

```diff
@@ -16933,7 +17005,8 @@
                 "81 1/2 Pa. 9",
                 "81* Pa. 57",
                 "81 * Pa. 57",
-                "*81 Pa. 57"
+                "*81 Pa. 57",
```

Comment:
Can we talk about the `*`s here.

Reply:
I don't know where that comes from, i'll try to find a citation like that.

Reply:
Did we find a citation here?

Reply:
This type of citations appear in "cite_to" key in the harvard cases, for example: https://api.case.law/v1/cases/555808/ or https://api.case.law/v1/cases/268550/

In total there are 730 citations with an *

Reply:
"cite": "*81 Pa. 57",
"cite": "81* Pa. 91",
"cite": "*81 Pa. 243",
"cite": "*81 Pa. 361",
"cite": "*81 Pa. 419",
"cite": "*81 Pa. 378",
"cite": "*81 Pa. 243",
"cite": "*81 Pa. 511",
"cite": "*81 Pa. 378",
"cite": "81* Pa. 166",
"cite": "*81 Pa. 41",
"cite": "*81 Pa. 243",
"cite": "*81 Pa. 497",
"cite": "81* Pa. 366",
"cite": "81* Pa. 19",

Reply:
It clearly says * in the pdfs, but our system doesnt ... if I remember ... doesnt allow special characters in the volume field.

Reply:
I think we can parse the citation volume around the *

Reply:
If i remember correctly the volume is stored in SmallIntegerField

what do you suggest we should do? reporters-db uses the regex only to validate the citation right and get the volume, reporter or page with the capture groups. We can't extract the volume without the * in reporters-db, that kind of behavior should go in eyecite, or maybe i'm missing something

Reply:
Just to make this full circle, after a discussion on slack, we think the * is an artifact from OCR and should be moved to opinionated, with the 1/2s as well.

Reply:
Understood, then i'll remove the support for * and 1/2
The * and 1/2 will be removed using opinionated fixes?